client/services.go

```diff
@@ -0,0 +1,154 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/systemd"
+)
+
+// ServicesOp encapsulate requests for performing an operation on a series of services
+type ServiceOp struct {
+	Services []string `json:"services,omitempty"`
+	Action   string   `json:"action"`
+}
+
+func (op ServiceOp) Description() string {
+	var verb string
+	switch op.Action {
+	case "enable-now":
+		verb = i18n.G("Enable and start")
+	case "disable-now":
+		verb = i18n.G("Stop and disable")
+	case "try-reload-or-restart":
+		verb = i18n.G("Try to reload or restart")
```

Comment:
These verbs above are not part of the interface we discussed, and it doesn't sound like a good idea to expose them right now. It feels like surfacing awkward parts of systemd instead of focusing on the functionality we want snapd to offer.